.github/workflows/publish-to-maven.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publishToMaven:
+    environment:
+      name: "MavenCentral"
+
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Enable KVM
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
+
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 17
+
+      - name: Make Gradle executable
+        run: chmod +x ./gradlew
```

Comment:
Why not check in this file as executable?

Reply:
It's already checked as, I'll remove this.
I'll also remove the KVM, it speeds up testing for connected android tests, but it's not relevant for build-release.